### PR TITLE
Update many dependency versions to latest that I could find

### PR DESCRIPTION
**Pull request by ByteLaw5**

### CODE REVIEW

**Comment by ByteLaw5** [comment]
This is crazy 😯😯😯

File: fabric/build.gradle [hunk 1]
```diff
@@ -77,18 +77,18 @@ processResources {
 
 shadowJar {
     configurations = [project.configurations.shadowCommon]
-    classifier "dev-shadow"
+    //classifier "dev-shadow"
```
Comment (QboiDev) [inline fabric/build.gradle]: This can be done using
```gradle
archiveClassifier.set("dev-shadow")
```

File: fabric/build.gradle [hunk 2]
```diff
@@ -77,18 +77,18 @@ processResources {
...
     injectAccessWidener = true
     input.set shadowJar.archiveFile
     dependsOn shadowJar
-    classifier null
+    //classifier null
 }
 
 jar {
-    classifier "dev"
+    //classifier "dev"
```
Comment (QboiDev) [inline fabric/build.gradle]: This can be done using
```gradle
archiveClassifier.set(null)
```

File: fabric/build.gradle [hunk 3]
```diff
@@ -77,18 +77,18 @@ processResources {
...
-    classifier "dev-shadow"
+    //classifier "dev-shadow"
 }
 
 remapJar {
     injectAccessWidener = true
     input.set shadowJar.archiveFile
     dependsOn shadowJar
-    classifier null
+    //classifier null
```
Comment (QboiDev) [inline fabric/build.gradle]: This can be done using
```gradle
archiveClassifier.set(null)
```

File: forge/build.gradle [hunk 4]
```diff
@@ -60,17 +60,17 @@ shadowJar {
...
 remapJar {
     input.set shadowJar.archiveFile
     dependsOn shadowJar
-    classifier null
+    //classifier null
 }
 
 jar {
-    classifier "dev"
+    //classifier "dev"
```
Comment (QboiDev) [inline forge/build.gradle]: This can be done using
```gradle
archiveClassifier.set("dev")
```

File: forge/build.gradle [hunk 5]
```diff
@@ -60,17 +60,17 @@ shadowJar {
...
     configurations = [project.configurations.shadowCommon]
-    classifier "dev-shadow"
+    //classifier "dev-shadow"
 }
 
 remapJar {
     input.set shadowJar.archiveFile
     dependsOn shadowJar
-    classifier null
+    //classifier null
```
Comment (QboiDev) [inline forge/build.gradle]: This can be done using
```gradle
archiveClassifier.set(null)
```

File: forge/build.gradle [hunk 6]
```diff
@@ -60,17 +60,17 @@ shadowJar {
     exclude "architectury.common.json"
 
     configurations = [project.configurations.shadowCommon]
-    classifier "dev-shadow"
+    //classifier "dev-shadow"
```
Comment (QboiDev) [inline forge/build.gradle]: This can be done using
```gradle
archiveClassifier.set("dev-shadow")
```

File: build.gradle [hunk 7]
```diff
@@ -1,8 +1,8 @@
 //file:noinspection GroovyAssignabilityCheck
 //file:noinspection GroovyAccessibility
 plugins {
-    id "architectury-plugin" version "3.4-SNAPSHOT"
-    id "dev.architectury.loom" version "1.0-SNAPSHOT" apply false
+    id "architectury-plugin" version "3.4.146"
+    id "dev.architectury.loom" version "1.2.341" apply false
```
Comment (QboiDev) [inline build.gradle]: `-SNAPSHOT` is recommended, as it will use the latest version.
Like `3.4-SNAPSHOT` and `1.2-SNAPSHOT`

**Review by QboiDev — CHANGES_REQUESTED** [review]
See comments

**Comment by QboiDev** [comment]
The classifiers are required.

**Review by QboiDev — APPROVED** [review]
Gimme those tasty commits

**Pull request merged**